src/daft-scan/src/lib.rs

```diff
@@ -451,6 +462,12 @@ impl ScanTask {
                 p2: sc2.pushdowns.clone(),
             });
         }
+        if sc1.file_path_column != sc2.file_path_column {
+            return Err(Error::DifferingFileFormatConfigsInScanTaskMerge {
```

Comment:
```suggestion
            return Err(Error::DifferingFilePathColumnsInScanTaskMerge {
```

Reply:
oops my bad, didn't realize the argument names were different. CI is failing on that :\

Reply:
yeah no worries, fixing